apps/server/src/infra/sync/tsp/tsp-sync.service.spec.ts

```diff
@@ -337,46 +339,49 @@ describe(TspSyncService.name, () => {
 		});
 	});
 
-	describe('findAccountByTspUid', () => {
+	describe('findAccountByExternalId', () => {
 		describe('when account is found', () => {
 			const setup = () => {
-				const tspUid = faker.string.alpha();
+				const externalId = faker.string.alpha();
+				const systemId = faker.string.alpha();
+
 				const user = userDoFactory.build();
 				const account = accountDoFactory.build();
 
-				user.id = tspUid;
+				user.id = faker.string.alpha();
+				user.externalId = externalId;
 				account.userId = user.id;
 
-				userService.findUsers.mockResolvedValueOnce({ data: [user], total: 1 });
+				userService.findByExternalId.mockResolvedValueOnce(user);
 				accountService.findByUserId.mockResolvedValueOnce(account);
```

Comment:
can be moved to main Describe no redundant code anymore

Reply:
I think this is not too redundant. These are only 2 test cases for this method.